src/plugins/ops.ts

```diff
@@ -236,6 +243,19 @@ export function addMutatingInfix(
   };
 }
 
+export const addPostfixIncAndDec: Plugin = {
+  name: "addPostfixIncAndDec",
+  visit(node) {
+    if (
+      node.kind === "MutatingInfix" &&
+      ["+", "-"].includes(node.name) &&
+      isIntLiteral(1n)(node.right)
```

Comment:
Note `x+=-1` and `x-=-1` work like `--x` and `++x` too.

Reply:
`mapToPrefixAndInfix` / `addMutatingInfix` never creates those.